src/main/java/racingcar/view/CarView.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.view;
+
+import racingcar.domain.Car;
+
+public class CarView {
+
+    public void printCarStatus(final Car car) {
+        StringBuilder carStatus = new StringBuilder(car.getName().value()).append(" : ");
+        carStatus.append(getCarProgressBarStatus(car.getPosition()));
```

Comment:
String.format을 사용해도 될 것 같아요 :) 
```suggestion
        StringBuilder carStatus = String.format("%s : %s", car.getName().value(), getCarProgressBarStatus(car.getPosition()));
```


그리고 `car.getNmae().value()`부분은 디미터의 법칙(디메테르의법칙)에대해 공부해보면 어떨까요? 링크 남겨둘게요! 
https://coding-start.tistory.com/264